tests/models/test_documents.py

```diff
@@ -182,6 +182,16 @@ def test_judgment_content_as_xml(self, mock_api_client):
             "test/1234", show_unpublished=True
         )
 
+    def test_judgment_content_as_xml_tree(self, mock_api_client):
+        mock_api_client.get_judgment_xml.return_value = (
+            '<?xml version="1.0" encoding="UTF-8"?><xml></xml>'
+        )
+        try:
```

Comment:
This is better written using [`with pytest.raises()`](https://docs.pytest.org/en/7.1.x/how-to/assert.html), which gives you a cleaner syntax.

Reply:
aha, I had a look at this, but in the case where we're testing that an invocation does **not** raise an exception, I couldn't see a way to use `pytest.raises`? There's an alternative with conditional raising [described here](https://stackoverflow.com/questions/20274987/how-to-use-pytest-to-check-that-error-is-not-raised) but to my eyes that adds complexity rather than removing it